commands/permissions.go

```diff
@@ -132,3 +177,54 @@ Scheme Managed: no
 
 	return nil
 }
+
+func assignUsersCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	role, response := c.GetRoleByName(args[0])
+	if response.Error != nil {
+		return response.Error
+	}
+
+	users := getUsersFromUserArgs(c, args[1:])
+
+	for _, user := range users {
+		startingRoles := strings.Fields(user.Roles)
+		for _, roleName := range startingRoles {
+			if roleName == role.Name {
+				return nil
+			}
+		}
+
+		userRoles := append(startingRoles, role.Name)
+		_, response = c.UpdateUserRoles(user.Id, strings.Join(userRoles, " "))
+		if response.Error != nil {
+			return response.Error
+		}
+	}
+
+	return nil
+}
+
+func unassignUsersCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	users := getUsersFromUserArgs(c, args[1:])
+
+	for _, user := range users {
```

Comment:
Same suggestion here:

```suggestion
	for i, user := range users {
		if user == nil {
			printer.PrintError("Couldn't find user '" + args[i+1] + "'")
			continue
		}
```